flow/workflows/cdc_flow.go

```diff
@@ -506,8 +506,8 @@ func CDCFlowWorkflow(
 			} else {
 				logger.Error("error in sync flow", slog.Any("error", err))
 			}
-			logger.Info("sync flow errored, sleeping for 10 minutes before retrying")
-			_ = workflow.Sleep(ctx, 10*time.Minute)
+			logger.Info("sync flow errored, sleeping for 30 seconds before retrying")
+			_ = workflow.Sleep(ctx, 30*time.Second)
```

Comment:
if this is issue then better fix can be worked out so that NumberOfSyncs isn't hitting error path